app/lib/features/chat/pages/room_page.dart

```diff
@@ -154,189 +153,182 @@ class _RoomPageConsumerState extends ConsumerState<RoomPage> {
   @override
   Widget build(BuildContext context) {
     final client = ref.watch(clientProvider);
+    final convo = ref.watch(currentConvoProvider);
     final chatRoomState = ref.watch(chatRoomProvider);
     ref.listen(messagesProvider, (previous, next) {
       if (next.isNotEmpty) {
         ref.watch(chatRoomProvider.notifier).isLoaded();
       }
     });
-    return OrientationBuilder(
-      builder: (context, orientation) => Scaffold(
-        backgroundColor: Theme.of(context).colorScheme.neutral,
-        resizeToAvoidBottomInset: orientation == Orientation.portrait,
-        appBar: AppBar(
-          backgroundColor: Theme.of(context).colorScheme.onPrimary,
-          elevation: 1,
-          centerTitle: true,
-          toolbarHeight: 70,
-          title: Column(
-            mainAxisSize: MainAxisSize.max,
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: [
-              Consumer(
-                builder: (context, ref, child) {
-                  final convoProfile = ref.watch(
-                    chatProfileDataProvider(widget.convo),
-                  );
-                  return convoProfile.when(
-                    data: (profile) {
-                      if (profile.displayName == null) {
-                        return Text(
-                          AppLocalizations.of(context)!.loadingName,
+    return convo != null
```

Comment:
this feels like bad error management. If, for whatever reason, there is no convo, we should show an error message - especially if that is mainly a developer error - rather than just showing nothing and covering up the broken assumption with an unclear blank screen...